.lintstagedrc.js

```diff
@@ -0,0 +1,10 @@
+const { CLIEngine } = require('eslint')
+
+const cli = new CLIEngine({})
+
+module.exports = {
+  '*.{js,json,css}': [
+    'prettier --write',
+    (files) => 'eslint --fix --max-warnings=0 ' + files.filter((file) => !cli.isPathIgnored(file)).join(' '),
```

Comment:
finally found a way for the lint staged to ignore package.json - nabbed from https://github.com/okonet/lint-staged#how-can-i-ignore-files-from-eslintignore-